### ISSUE TITLE
⌨ Missing labels on inputs

### ISSUE THREAD

**Issue by BrookeCameron** [GCTC-NTGC/gc-digital-talent#16026]
### fable tester reported: 
While creating a process the dropdown menu items for "Group and level", "Community", and "Department", are not properly labeled by my Assistive Technology, preventing use of my primary navigation approach. I can complete the required steps using alternate methods; however, this adds friction and decreases efficiency when multiple dropdowns are present. I need to resort to voice commands simulating keyboard navigation or rely on eye tracking to choose dropdown items.

### Expected behaviour: 
Drop downs are labeled properly

### 📋 Steps to Reproduce

1. sign in 
2. switch to community view 
3. navigate to processes 
4. create a new process 
5. fill out the first section of the process (group and level, community and department) 

### 🕵️ Details

3.3.2 

### What operating systems are you encountering the problem on?

Windows

### What browsers are you encountering the problem on?

Chrome

### What assistive technologies are you encountering the problem on?

Talon Voice

### ✅ Acceptance Criteria

- [ ] inputs are properly labeled
